OpenAPI Update for 10.02.130

hurco-satterwhitej opened the pull request:
Adds OpenAPI schemas for new REST bulks: BulkMillNCVariable, BulkLoadedPrograms, and BulkActualTopology.

hurco-satterwhitej committed:
OpenAPI Update for 10.02.130

Adds OpenAPI schemas for new REST bulks: BulkMillNCVariable, BulkLoadedPrograms, and BulkActualTopology.

hurco-robertsi commented on openapi.json:
```diff
@@ -1288,6 +1339,134 @@
+          }
+        }
+      },
+      "BulkActualTopology": {
```
we shouldn't expose this

hurco-robertsi commented on openapi.json:
```diff
@@ -1288,6 +1339,134 @@
+          }
+        }
+      },
+      "ActNetworkNodeType": {
```
or this

hurco-satterwhitej committed:
Remove openapi object exposure

Anticipated two additional objects to expose via openapi while adding to the file that shouldn't have been exposed.

hurco-satterwhitej committed:
Remove box
